test/com/ableton/JenkinsMocks.groovy

```diff
@@ -9,4 +9,12 @@ class JenkinsMocks {
   static Closure isUnix = {
     return !System.properties['os.name'].toLowerCase().contains('windows')
   }
+
+  static Closure pwd = { args ->
+    if (args?.tmp) {
+      return System.properties['java.io.tmpdir']
+    } else {
```

Comment:
I am not a fan of `return` after `else`.

Reply:
Actually we can refactor to this one-liner:

```groovy
    return args?.tmp ? System.properties['java.io.tmpdir'] : System.properties['user.dir']
```

Reply:
Of course :sweat_smile:

Reply:
Actually: `return System.properties[args.?tmp ? 'java.io.tmpdir' : 'user.dir']